pyogrio/tests/test_arrow.py

```diff
@@ -817,7 +847,7 @@ def test_write_memory(naturalearth_lowres, driver):
 
     buffer = BytesIO()
     write_arrow(
-        table.slice(0, 1),
+        table,
```

Comment:
The reason I did this was to avoid the filterwarning you added .. ;) Ideally we would add a test dataset without mixed geometries that we can use for most tests .. (or just edit the naturalearth fixture to give all MultiPolygons), but that's for another issue

Reply:
I switched the places where we were doing `.slice(0, 1)` to only read 1 feature from naturalearth, and sidestep the issue (and also still allow comparison of first table to the written & reread table).